pkg/kubelet/volumes.go

```diff
@@ -285,10 +285,6 @@ func (kl *Kubelet) newVolumeAttacherFromPlugins(spec *volume.Spec, pod *api.Pod,
 	if err != nil {
 		return nil, fmt.Errorf("can't use volume plugins for %s: %v", spec.Name(), err)
 	}
-	if plugin == nil {
```

Comment:
I don't know how I cross reference morie's comment.  he's got it right.  

@swagiaal i agree with you.